internal/pkg/earpointer/analysis_test.go

```diff
@@ -625,13 +625,6 @@ func TestMakeInterface(t *testing.T) {
 func TestUnimplemented(t *testing.T) {
 	code := `package p
 	func f() {
-		defer func() {
-			print("defer")
-		}()
-		go func() {
-			print("go")
-		}()
-
```

Comment:
Is this `Test*` just a scratch pad for the things you are still intending to cover?  This block of deletion caught me off guard, and I note tests still pass with or without these lines.

Reply:
They are now covered by "TestGoDefer(t *testing.T)". 
Previously I added them purely to obtain better code coverage.